### PR TITLE
Sorted by value and added Arceus' forms

### PR DESCRIPTION
**Pull request by AsparagusEduardo**

### CODE REVIEW

File: structs/pokemon.h [hunk 1]
```diff
@@ -89,7 +96,9 @@ typedef enum {
     UNOWN_W=176,
     UNOWN_X=184,
     UNOWN_Y=192,
-    UNOWN_Z=200
+    UNOWN_Z=200,
+    UNOWN_EXCLAMATIONMARK =208,
```
Comment (martmists) [inline structs/pokemon.h:100]: I know I initially wrote these, but maybe add an underscore before `MARK`. Also (probably my bad too), the spaces are inconsistent. We should put 1 space before and 1 space after the equals sign.

Comment (AsparagusEduardo) [reply structs/pokemon.h:100]: Should I also align the equal signs so it looks better?
```
typedef enum {
    ARCEUS_NORMAL   = 0,
    ARCEUS_FIST     = 8,
    ARCEUS_SKY      = 16,
```

Comment (martmists) [reply structs/pokemon.h:100]: No, just use 1 space on either side of the equals signs.

Comment (AsparagusEduardo) [reply structs/pokemon.h]: Done :)

**Comment by AsparagusEduardo** [comment]
I just remembered that type ???-type Arceus was supposedly present in Gen IV data. Where should that one go in relation to the other ones? (Since it's order is not clear in the Bulbapedia page)

**Comment by martmists** [comment]
Leaving it out is fine, if it's part of a failsafe we'll figure it out later on.

**Pull request merged**